src/impl.zig

```diff
@@ -78,7 +78,14 @@ pub fn disasmIterManaged(handle: Handle, code: []const u8, address: u64) IterMan
 /// Return an Iter object
 /// Does not yet consume any element.
 pub fn disasmIter(handle: Handle, code: []const u8, address: u64, ins: [*]insn.Insn) Iter {
-    return Iter.init(handle, code, address, ins);
+    return Iter{
+        .handle = handle,
+        .code = code,
+        .original_code = code,
+        .original_address = address,
+        .address = address,
+        .insn = ins,
+    };
 }
```

Comment:
Now this whole function seems to be unnecessary.
This might be a style-question, but simply constructing structures by the initialization form seems in my opinion a bit better, considering that a init function that doesn't really do much other than fill in parameters seems a bit superfluous.